### PR TITLE
add additional e2e rbac bindings to match existing users

### PR DESCRIPTION
**Pull request by deads2k**
Adds RBAC bindings that are particular to the users and groups for e2e tests to prime the authorizer.

### CODE REVIEW

**Comment by deads2k** [comment]
@cjcullen @mikedanese This is ready for review, last commit only (the rest is https://github.com/kubernetes/kubernetes/pull/38626).

**Comment by deads2k** [comment]
rebased on https://github.com/kubernetes/kubernetes/pull/38626 .  Still just the last commit to look at.

**Comment by deads2k** [comment]
@k8s-bot unit test this: issue #37982

File: cluster/addons/e2e-rbac-bindings/admin-binding.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+kind: ClusterRoleBinding
+metadata:
+  name: admin-cluster-admin
+  labels:
+    kubernetes.io/cluster-service: "true"
```
Comment (liggitt) [inline cluster/addons/e2e-rbac-bindings/admin-binding.yaml]: what does this annotation do?

File: cluster/addons/e2e-rbac-bindings/admin-binding.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: view
+subjects:
+- apiVersion: rbac/v1alpha1
+  kind: User
+  name: admin
```
Comment (liggitt) [inline cluster/addons/e2e-rbac-bindings/admin-binding.yaml]: a comment in these files about what they are needed for, and whether they are an acceptable end state or have a TODO to be removed/tightened might be helpful

**Review by liggitt — COMMENTED** [review]
are the RBAC bindings largely mirroring existing ABAC permissions?

File: cluster/addons/e2e-rbac-bindings/admin-binding.yaml (hunk 1 above)
Comment (deads2k) [inline cluster/addons/e2e-rbac-bindings/admin-binding.yaml]: > what does this annotation do?

kube addon manager filters on it for deciding what to create.

**Comment by deads2k** [comment]
comments addressed.

**Pull request merged**